CoreVulkanDisplay/src/au/gov/asd/tac/constellation/visual/vulkan/renderables/CVKIconsRenderable.java

```diff
@@ -1577,14 +1622,14 @@ private int CreatePipelineLayout() {
         return ret;        
     }    
     
-    private int CreatePipelines() {
+    private int CreatePipelines(long renderPassHandle, List<Long> pipelinesList) {
```

Comment:
We probably need a better name than pipelinesList.  What makes this different from pipelines?

Reply:
Oh I see.  Maybe pipelines needs to be renamed presentationPipelines or something?

Reply:
no difference, just passed in as a param

Reply:
Yeah I think the variable pipelines needs to be renamed displayPipelines and the other should be hitTestPipelines.

Reply:
renamed all three